### PR TITLE
Add release 1.11.4

### PR DESCRIPTION
**Pull request by godfreyhe**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -106,6 +106,37 @@ flink_releases:
...
+        url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.4/flink-1.11.4-bin-scala_2.12.tgz"
+        asc_url: "https://downloads.apache.org/flink/flink-1.11.4/flink-1.11.4-bin-scala_2.12.tgz.asc"
+        sha512_url: "https://downloads.apache.org/flink/flink-1.11.4/flink-1.11.4-bin-scala_2.12.tgz.sha512"
+    source_release:
+      name: "Apache Flink 1.11.4"
+      id: "1114-download-source"
+      url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.11.4/flink-1.11.4-src.tgz"
+      asc_url: "https://downloads.apache.org/flink/flink-1.11.4/flink-1.11.4-src.tgz.asc"
+      sha512_url: "https://downloads.apache.org/flink/flink-1.11.4/flink-1.11.4-src.tgz.sha512"
+    release_notes_url: "https://ci.apache.org/projects/flink/flink-docs-release-1.12/release-notes/flink-1.12.html"
```
Comment (Myasuka) [inline _config.yml]: The release notes url is not correct.

**Review by Myasuka — COMMENTED** [review]
Thanks for creating this PR, please take a look at my commnets.

File: _posts/2021-07-26-release-1.11.4.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+categories: news
+authors:
+- godfrey:
+  name: "Xiaoling He"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.11 series.
+
+This release includes 56 fixes and minor improvements for Flink 1.11.4. The list below includes a detailed list of all fixes and improvements.
```
Comment (leonardBang) [inline _posts/2021-07-26-release-1.11.4.md]: `1.11.4`-> `1.11.3`

Comment (carp84) [reply _posts/2021-07-26-release-1.11.4.md]: The number of issues resolved in 1.11.4 should be 78 according to the [release note](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12349404).

File: _config.yml (hunk 1 above)
Comment (carp84) [reply _config.yml]: Indeed, we should correct this link.

**Review by carp84 — CHANGES_REQUESTED** [review]
Please resolve the comments before merging, and update the release date in both `config.yml` and post file name accordingly.

**Comment by godfreyhe** [comment]
@carp84 @Myasuka @leonardBang Thanks for the review. I will update the post file name later

**Comment by godfreyhe** [comment]
@carp84  please take another look, thanks

File: _config.yml [hunk 3]
```diff
@@ -189,6 +219,10 @@ release_archive:
         version_short: "1.12"
         version_long: 1.12.0
         release_date: 2020-12-08
+      -
+        version_short: "1.11"
+        version_long: 1.11.4
+        release_date: 2021-07-26
```
Comment (carp84) [inline _config.yml]: Minor: we need to update the release date accordingly.

**Review by carp84 — APPROVED** [review]
+1, left a minor comment.

**Pull request closed without merging**